pandas/_libs/tslibs/conversion.pyx

```diff
@@ -393,6 +393,25 @@ cdef _TSObject convert_datetime_to_tsobject(datetime ts, object tz,
     return obj
 
 
+cdef _TSObject setup_tsobject_tz_using_offset(_TSObject obj,
```

Comment:
call this create_tsobject_tz_using_offset

Reply:
renamed